assets/optional/multus/05-configmap.yaml

```diff
@@ -14,7 +14,7 @@ data:
     {
         echo "$(date --iso-8601=seconds) [cnibincopy] ${1}"
     }
-    DESTINATION_DIRECTORY=/host/opt/cni/bin/
+    DESTINATION_DIRECTORY=/host/var/lib/cni/bin/
```

Comment:
What happens if we upgrade in place to a version that includes this change? Will we end up with the old binaries in the old location on the host?

Reply:
We have a preStop hook. So the files will be cleaned when the old multus pod is terminated.

Reply:
Good, thanks, @pliurh !